scout/dashboard.md

```diff
@@ -29,6 +29,11 @@ Each entry in the list shows the following details:
 - The most recent version of the image and the vulnerabilities for that version. Clicking the link for the base image opens [the image layer view](#image-layer-view).
 - The operating system and architecture of the image.
 - The base image and version used by the repository and the vulnerabilities for that version. Clicking the link for the base image opens [the image layer view](#image-layer-view).
+
+  > **Note**
+  >
+  > Docker Scout detects the base image used by a repository by using the digest hash. However, if the hash is linked to different tags, Docker Scout can't show the precise version.
```

Comment:
?

```suggestion
  > Docker Scout detects the base image used by a repository by using the digest hash. However, if the hash is associated with more than one tag, Docker Scout can't show the precise version.
```

Reply:
@dvdksn Thanks, I was stuck on that particular word for a while, perfect.